content/en/docs/collector/registry.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Registry
+linkTitle: Registry
+description:
+  Exporters, processors, receivers and other useful components for the
+  OpenTelemetry Collector
+redirect: https://opentelemetry.io/ecosystem/registry/?language=collector
+manualLinkTarget: _blank
```

Comment:
Remove `manualLinkTarget` from here and the other files, because this isn't an external link:

```suggestion
```